### PR TITLE
Adding examples of PUT requests for VAPI

### PR DESCRIPTION
**Pull request by tommorris**
For some reason, this didn't make it onto GitHub before.

### CODE REVIEW

File: _examples/api/voice/calls/put-request/transfer.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+title: Transfer
+menu_weight: 6
+---
+
+```json
+{
+    "action": "transfer",
+    "destination": {
+      "type": "ncco",
+      "url:" ["http://example.org/new-ncco.json"]
```
Comment (adambutler) [inline _examples/api/voice/calls/put-request/transfer.md]: Invalid JSON, should be:

```
"url": ["http://example.org/new-ncco.json"]
```

Comment (tommorris) [reply _examples/api/voice/calls/put-request/transfer.md]: @adambutler fixed.

**Comment by tommorris** [comment]
The content was supposed to have been added in #222.

**Comment by leggetter** [comment]
@tommorris I know this has been merged - a small request: can we try to make everything linkable?

I don't know if we can link to the examples with the transfer tab selected. But maybe we could put an `### Examples` heading which would mean we could say `Go [here] and click "Transfer" to see the example`.

**Pull request merged**